docker-traceability-api/pom.xml

```diff
@@ -132,6 +73,20 @@
       <groupId>com.github.docker-java</groupId>
       <artifactId>docker-java</artifactId>
       <version>${docker-java.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.apache.httpcomponents</groupId>
+          <artifactId>httpclient</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>org.apache.httpcomponents</groupId>
+          <artifactId>httpmime</artifactId>
+        </exclusion>
```

Comment:
Exclude `httpclient` and `httpmime` since `docker-java` brings versions that are not compatible with HtmlUnit when run against 2.7 baseline.